src/mame/drivers/kaneko16.cpp

```diff
@@ -160,6 +160,16 @@ WRITE16_MEMBER(kaneko16_state::kaneko16_coin_lockout_w)
 
 ***************************************************************************/
 
+void kaneko16_state::kaneko16_common_oki_bank_w(int bankno, int bank, size_t fixedsize, size_t bankedsize)
+{
+	size_t length = (m_okiregion[bankno]->bytes() - fixedsize) / bankedsize;
+
+	if (bank < length)
+	{
+		m_okibank[bankno]->set_enrty(bank);
```

Comment:
You should at least be trying to compile your own changes before submitting pull requests filled with obvious compilation errors like this typo.